### ISSUE TITLE
Relationship field - Additional Field Options

### ISSUE THREAD

**Issue by StaggerLeee** [pods-framework/pods#2808]
Can we get some more detailed information/tutorials (for beginners) about those options ?
Possibly with screenshots or screencast. 
As it is now I have no clue what to put there, why to put there, and what to expect as output in Metabox.

Pods is very powerful but I am afraid that because of to tehnical documentations, or in some parts lacking more human documentation I am using it only for 10% it could and should be used.

As it is now all I do with Pods I can do any other fields set, or ACF for instance. And i read and read forums and documentation. 

Right now have several Custom Post Types and custom Taxonomies I did manualy in functions.php and in addition with ACF. Playing with Pods to replace all of it, but dont see any benefit.

I repeat, problem is in me, not Pods. I read and read but documentation is to tehnical and complicated.
This that it is easy to understand are precise those thing I can do with other addons. So in some way it is a shame. So much under the hood, so powerful and yet not used even 10%.

**Comment by Shelob9** [comment]
@StaggerLeee Honestly, the additional field options in relationship field settings are not for beginners. I can't write a tutorial on those without assuming a basic understanding of how a WHERE clause works in MySQL, which I think clases it beyond "beginner WordPress tutorial."

I would encourage you, if you're struggling with Pods to join us in the #support channel of our Slack (https://Pods.io/chat) where our team and helpful members of our community are happy to answer your questions.

Also, you may wish to check out [Caldera Easy Pods](https://calderawp.com/downloads/caldera-easy-pods/) -- it is a commercial, third-party plugin that makes creating Pods queries, and showing them via Pods Templates much easier.

**Comment by StaggerLeee** [comment]
I dont have problem with templates, but Admin Metaboxes. (not now anyway, it was not easy)
What about " Display Field in Selection List" ? 
Your documentation for this is one sentence long.

**Comment by Shelob9** [comment]
Yes, that should link to http://pods.io/docs/build/special-magic-tags/ where it is explained better.

**Comment by StaggerLeee** [comment]
Never mind, i guess it is as it is.
Dont see any purpose of almost all those magic tags in relationship select/dropdown.

Make some more real life examples like User directory.

**Comment by Shelob9** [comment]
http://pods.io/tutorials/using-pods-create-user-directory/

**Comment by StaggerLeee** [comment]
Thank you man, i understand your position. :)
If I ever manage Pods on very advanced level would you accept my screencast tutorials for documanetation ?
Some real life examples.
Without sound of course, dont want to torture people with my english accent.

**Comment by sc0ttkclark** [comment]
You have an english accent? Don't mute yourself, let it shine!

http://images5.fanpop.com/image/photos/31400000/When-i-first-saw-this-movie-in-was-all-let-it-shine-31437501-178-120.gif

**Comment by StaggerLeee** [comment]
Missing "awful" in the front of "english".

**Comment by sc0ttkclark** [comment]
How is your english accent awful? I require proof of all such claims

**Comment by Shelob9** [comment]
@StaggerLeee We would absolutely love to have screencasts. They don't have to be advanced. In fact, I think a new user's perspective could be really helpful. Anything you can share about how you used Pods to accomplish something cool with WordPress would be awesome to have.

Please let me know how I can help.
